.github/workflows/macos-check.yaml

```
@@ -0,0 +1,97 @@
+name: macOS Check
+on:
+  workflow_dispatch: # Manual trigger
+  pull_request:
+    paths-ignore:
+      - .gitignore
+      - .github/**
+      - '!.github/workflows/macos-check.yaml'  # Run check on self change
+      - CONTRIBUTORS
+      - LICENSE
+      - NOTICE
+      - README.md
+      - android/**
+      - iphone/**
+      - data/strings/**
+      - docs/**
+      - packaging/**
+      - pyhelpers/**
+      - tools/**
+      - '!tools/python/run_desktop_tests.py'
+      - '!tools/python/testserver.py'
+      - '!tools/python/SiblingKiller.py'
+      - xcode/**
+
+jobs:
+  macos-matrix:
+    name: macOS builds and tests
+    runs-on: macos-12
+    env:
+      HOMEBREW_NO_ANALYTICS: 1
+      HOMEBREW_NO_INSTALL_CLEANUP: 1
+    strategy:
+      fail-fast: false
+      matrix:
+        CMAKE_BUILD_TYPE: [Debug, RelWithDebInfo]
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+
+      - name: Parallel submodules checkout
+        shell: bash
+        run: git submodule update --depth 1 --init --recursive --jobs=$(($(sysctl -n hw.logicalcpu) * 20))
+
+      - name: Install build tools and dependencies
+        shell: bash
+        run: |
+          brew install ninja qt@5
+
+      - name: Configure
+        shell: bash
+        run: ./configure.sh
+
+      - name: CMake
+        shell: bash
+        run: |
+          echo "Building ${{ matrix.CMAKE_BUILD_TYPE }}"
+          cmake . -B build -G Ninja -DCMAKE_BUILD_TYPE=${{ matrix.CMAKE_BUILD_TYPE }} -DCMAKE_C_FLAGS=-g1 -DCMAKE_CXX_FLAGS=-g1
+
+      - name: Compile
+        shell: bash
+        working-directory: build
+        run: ninja
+
+      - name: Checkout world_feed_integration_tests_data
+        uses: actions/checkout@v3
+        with:
+          repository: omapsapp/world_feed_integration_tests_data
```

```suggestion
          repository: organicmaps/world_feed_integration_tests_data
```

Should also be fixed for the Linux check.

---

Changed in both.